build/cli.version.darwin

```diff
@@ -1 +0,0 @@
-1.0.0-preview2-1-003167
```

Comment:
The CLI has been producing builds on all platforms pretty consistently. We no longer need these to be jagged.